style_editor.html

```diff
@@ -7165,6 +7171,46 @@
   power_button.style.backgroundColor = STATE_ON ? "lightblue" : "";
   console.log("POWER");
   blade.addEffect(STATE_ON ? EFFECT_IGNITION : EFFECT_RETRACTION, Math.random() * 0.7 + 0.2);
+  ToggleButtons();
+}
+
+function ToggleButtons() {
+  var preon_button = FIND("PREON_BUTTON");
+  var blast_button = FIND("BLAST_BUTTON");
+  var clash_button = FIND("CLASH_BUTTON");
+  var stab_button = FIND("STAB_BUTTON");
+  var force_button = FIND("FORCE_BUTTON");
+  var lockup = FIND("LOCKUP");
+  var drag_button = FIND("DRAG_BUTTON");
+  var melt_button = FIND("MELT_BUTTON");
+  var lb_button = FIND("LB_BUTTON");
+  var boot_button = FIND("BOOT_BUTTON");
+  var newfont_button = FIND("NEWFONT_BUTTON");
+  var battery_button = FIND("BATTERY_BUTTON");
+  var powersave_button = FIND("POWERSAVE_BUTTON");
+  if (STATE_ON) {
```

Comment:
I think it would be better to just do:

FIND("PREON_BUTTON").disabled = !!STATE_ON;
and for the ones that should be off:
FIND("BLAST_BUTTON").disabled = !STATE_ON;